text/0507-subnets.md

```diff
@@ -0,0 +1,402 @@
+# Full control over VPC and subnet configuration
+
+* **Original Author(s):**: @otaviomacedo
+* **Tracking Issue**: #507
+* **API Bar Raiser**: @{BAR_RAISER_USER}
+
+New L2 constructs to allow for greater control over VPC design. Among other
+things, these constructs allow users to create and configure subnets in
+ways that are not currently possible with the `Vpc` construct.
+
+## Working Backwards
+
+### IP addressing
+
+When you create a VPC, in addition to the primary IP block, you can provide
+one or more secondary IP blocks, by providing them in the CIDR format:
+
+```ts
+const vpc = new Vpc(this, 'vpc', {
```

Comment:
Honestly I think we can also go `VpcV2`.

Reply:
I've changed my mind about this, as well. I think it's safer to keep them separate. Doc updated.